doc/GIT.md

```diff
@@ -147,13 +144,10 @@ After being added to whitelist you can trigger buildjobs by saying:
 * jenkins build [multiconfig-gcc-stable](https://build.libelektra.org/job/elektra-multiconfig-gcc-stable/) please
 * jenkins build [multiconfig-gcc47-cmake-options](https://build.libelektra.org/job/elektra-multiconfig-gcc47-cmake-options/) please
 * jenkins build [source-package-test](https://build.libelektra.org/job/elektra-source-package-test/) please
-* jenkins build [stable](https://build.libelektra.org/job/elektra-mergerequests-stable/) please
-* jenkins build [unstable](https://build.libelektra.org/job/elektra-mergerequests-unstable/) please
 * jenkins build [homepage](https://build.libelektra.org/job/elektra-homepage/) please
-* jenkins build [gcc-configure-debian-stretch-minimal](https://build.libelektra.org/job/elektra-gcc-configure-debian-stretch-minimal/) please
-* jenkins build [gcc-configure-debian-jessie-minimal](https://build.libelektra.org/job/elektra-gcc-configure-debian-jessie-minimal/) please
-* jenkins build [docker](https://build.libelektra.org/job/test-docker/) please
-* jenkins build [haskell](https://build.libelektra.org/job/elektra-haskell/) please
+* jenkins build [jenkinsfile](https://build.libelektra.org/jenkins/job/elektra-jenkinsfile/)[REGEX] please
+  * Use the optional `REGEX` field to specify which stages you want to run
+    The REGEX has to match the stagename.
```

Comment:
Can you move the deleted jobs as sub-items here? These names can be used for the regex, can't they?

Reply:
no they can not

Reply:
` The REGEX has to match the stagename.`  should indicate that you are to use the stage names of the jenkins file which can be easily read from the webui

Reply:
Can you please add a link to the webui then?

Reply:
it is literally there in line 148, even clickable...